Session.js

```diff
@@ -0,0 +1,56 @@
+class Session { 
+// To be utilized when unwrapping JSON from the future servlets. Will be on the 
```

Comment:
put these comments above line 1 to be consistent with attendee class and then remove empty line at line 5